Remotion/Web/Development.WebTesting.IntegrationTests/TabbedMultiViewControlObjectTest.cs

```diff
@@ -116,26 +124,43 @@ public void TestSwitchTo ()
       tabbedMultiView = home.TabbedMultiViews().Single();
       Assert.That (tabbedMultiView.Scope.Text, Is.StringContaining ("Content2"));
       Assert.That (tabbedMultiView.Scope.Text, Is.Not.StringContaining ("Content1"));
+      Assert.That (tabbedMultiView.Scope.Text, Is.Not.StringContaining ("Content3"));

```

Comment:
why do we need the tab3 tests here?

Reply:
It looks like a tab with an icon was added to test its accessibility. I'm guessing we added this to this test to keep it "complete"

Reply:
kill it in this test. It feels redundant to do same thing 3 times.